src/views/Rules/ListRules/index.jsx

```diff
@@ -249,9 +297,35 @@ function ListRules(props) {
       throw error;
     }
 
-    setRulesWithScheduledChanges(
-      rulesWithScheduledChanges.filter(i => i.rule_id !== dialogRule.rule_id)
-    );
+    if (Object.keys(dialogRule.requiredSignoffs).length > 0) {
+      // A change was scheduled, we need to update the card
+      // to reflect that.
+      try {
```

Comment:
No need to wrap in try catch if we make use of the `onError` and `onComplete` props of the dialog.